src/braze/tags/connectedContent.ts

```diff
@@ -18,11 +20,19 @@ export default <ITagImplOptions>{
     const options = match[2]
     this.options = {}
     if (options) {
-      options.split(/\s*:/).forEach((optStr) => {
```

Comment:
Are the contextual variables populated by this point? If not, the lazy search for the headers JSON will encounter a closing `}` inside the header JSON that is the end of that variable (i.e. `{{userId}}`) and isn't the end of the JSON block.

Reply:
This is actually something that I will need to do as well.

Reply:
@yq314 I updated the Regex to allow those `{{attributes}}` to not break the lazy search in the headers regex, and also allowed them in the body option.

Reply:
looks good 👍